packages/flutter_tools/lib/src/ios/device_ios.dart

```diff
@@ -235,7 +235,7 @@ class IOSDevice extends Device {
       return 2;
     }
     return await runCommandAndStreamOutput([loggerPath],
-        prefix: 'iOS dev: ', filter: new RegExp(r'.*SkyShell.*'));
+        prefix: 'iOS: ', filter: new RegExp(r'.*FlutterRunner.*'));
```

Comment:
The `.*` shouldn't be needed, it's implicit in a regular expression. And if your regular expression is just a string literal, you can replace it with just a string. (Assuming this is being used as a general match and not an anchored match.)